Common/Data/Consolidators/WickedRenkoConsolidator.cs

```diff
@@ -0,0 +1,272 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using QuantConnect.Data.Market;
+
+namespace QuantConnect.Data.Consolidators
+{
+    /// <summary>
+    /// This consolidator can transform a stream of <see cref="BaseData"/> instances into a stream of <see cref="RenkoBar"/>
+    /// with Renko type <see cref="RenkoType.Wicked"/>.
+    /// </summary>
+    public class WickedRenkoConsolidator : IDataConsolidator
+    {
+        internal DateTime CloseOn;
+        internal decimal CloseRate;
+        internal decimal HighRate;
+        internal decimal LowRate;
+        internal DateTime OpenOn;
+        internal decimal OpenRate;
+        internal decimal BarSize;
```

Comment:
Let's please make these protected and create a test class `WickedRenkoConsolidatorTest` exposing them so they can be asserted on 👍

Reply:
Alternatively, changing them to protected internals will expose them without needing to re-add something of the form of the removed `OpenRenkoBar`. Would that be preferable?